Ushindi-Zirimwabagabo-Josue/Gravity-House
# #172914703 home page

What does this PR do?
create a home page

Description of Task to be completed?
-create the HTML markup
-add style to the markup
-add the bootstrap files

How should this be manually tested?
-make sure you are in your virtual environment
-run the server
-open a web browser
-enter the server's address

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
[#172914703]https://www.pivotaltracker.com/story/show/172914703

Screenshots (if appropriate)
N/A

Questions:
N/A
